examples/gaussian_processes/gaussian_process.ipynb

```diff
@@ -4,18 +4,23 @@
    "cell_type": "markdown",
```

Comment:
<p>Finally, update this cell to be:</p><p> </p><pre class="ql-syntax" spellcheck="false">_, ax = plt.subplots(figsize=())
ax.plot(xgrid, y_pred.transpose(..., "sample"), c-", alpha=0.1)
ax.plot(x, y, "r.");
</pre><p> </p><p>Note: the figsize is so the plot becomes a bit bigger (as it used to be in the old notebook) which I think looks nicer, I don't know the figsize that should be used though. If the current smaller one was preferred it could be simplified to</p><p> </p><pre class="ql-syntax" spellcheck="false">plt.plot(xgrid, y_pred.transpose(..., "sample"), c-", alpha=0.1)
plt.plot(x, y, "r.");
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>